### DISCUSSION TITLE
Lazy loadPrevNext not working in react.

### DISCUSSION THREAD

**Discussion by denny7** [nolimits4web/swiper#4325] [Q&A]
Hello,
I am trying to implement swiper with lazy loading. When I use `lazy={true}` it is working, but when I try to add loadPrevNext all images are downloaded. My code is: 

```
import SwiperCore, {Navigation, Pagination, Lazy} from 'swiper';
import {Swiper, SwiperSlide} from 'swiper/react';

SwiperCore.use([Navigation, Pagination, Lazy]);

<Swiper
        navigation
        pagination={{clickable: true}}
        threshold={6}
        preloadImages={false}
        watchSlidesVisibility={true}
        lazy={{
            loadPrevNext: true,
            loadOnTransitionStart: true
        }}
    >
...
</Swiper>
```

Is there any bug or I am doing it wrong?

**Comment by asdutoit** [comment]
Hi @denny7 , did you ever manage to get your issue resolved?   I have a similar issue using it with Nextjs

**Reply by denny7** [reply]
Hello @asdutoit! Currently I'm using swiper 7.0.6. and next 11.1.2 and it seems everything is working well.

**Reply by asdutoit** [reply]
I am prob just being a tool.   There is most likely a spelling mistake in the code which I overlooked.   
Thanks for the response!
